notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>I think we should give a bit more guidance on why or when to use this (of course can't go in detail, that is for the user guide), but now both the "column names" and "condition expression" have already been used above without using loc. So why use loc here? </p><p> </p><p>The main difference is maybe that here you are selecting rows and columns in one go?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I agree with @jorisvandenbossche. At minimum, I would explain that the part before the comma is the rows you want, and the part after the comma is the columns you want.</p><p> </p><p>I might also mention the options for what you can pass it: a single label, a list of labels, a slice of labels, a conditional expression, or a colon.</p>